docs/Operators.md

```diff
@@ -3903,7 +3903,7 @@ node = onnx.helper.make_node(
     outputs=['y']
 )
 x = np.array([1., 2., 3., 4., 5.]).astype(np.float64)
-axis = np.array([0]).astype(np.int32)
+axis = np.array(0).astype(np.int32)
```

Comment:
Why are we removing the array brackets here? 

According to numpy docs this argument should be an array or an object exposing the array interface:
https://numpy.org/doc/stable/reference/generated/numpy.array.html

Reply:
The CumSum specification documentation states that `axis` is a 0-D tensor (scalar). We are removing the array brackets so `axis` isn't a 1-D tensor.

```
>>> axis = np.array([0]).astype(np.int32)
>>> axis.ndim
1
>>> axis = np.array(0).astype(np.int32)
>>> axis.ndim
0
```

Reply:
numpy array scalars https://numpy.org/doc/stable/reference/arrays.scalars.html